src/spells/RwaSpellGoerli.sol

```diff
@@ -0,0 +1,242 @@
+        CHANGELOG.setAddress("RWA008SGHWOFH1_A_OUTPUT_CONDUIT", RWA008SGHWOFH1_A_OUTPUT_CONDUIT);
+
+        /// @notice bump changelog version
+        // TODO make sure to update this version on mainnet
+        CHANGELOG.setVersion("1.0.0");
+
+        /// @notice Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA008SGHWOFH1_A).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008SGHWOFH1_A).ilk() == ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA008SGHWOFH1_A).gem() == RWA008SGHWOFH1_GEM, "join-gem-not-match");
+        require(
+            GemJoinAbstract(MCD_JOIN_RWA008SGHWOFH1_A).dec() == DSTokenAbstract(RWA008SGHWOFH1_GEM).decimals(),
+            "join-dec-not-match"
+        );
+
+        /**
+         * @notice init the RwaLiquidationOracle
+         * doc: "doc"
+         * tau: 5 minutes
+         */
+        // TODO: this should be verified with RWA Team (5 min for testing is good)
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA008SGHWOFH1_A_INITIAL_PRICE, DOC, 300);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+        CHANGELOG.setAddress("PIP_RWA008SGHWOFH1", pip);
+
+        /// @notice Set price feed for RWA008SGHWOFH1
+        SpotAbstract(MCD_SPOT).file(ilk, "pip", pip);
+
+        /// @notice Init RWA008SGHWOFH1 in Vat
+        VatAbstract(MCD_VAT).init(ilk);
+        /// @notice Init RWA008SGHWOFH1 in Jug
+        JugAbstract(MCD_JUG).init(ilk);
+
+        /// @notice Allow RWA008SGHWOFH1 Join to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RWA008SGHWOFH1_A);
+
+        /// @notice Allow RwaLiquidationOracle to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MIP21_LIQUIDATION_ORACLE);
+
+        /// @notice 1000 debt ceiling
+        VatAbstract(MCD_VAT).file(ilk, "line", RWA008SGHWOFH1_A_INITIAL_PRICE);
+        VatAbstract(MCD_VAT).file("Line", VatAbstract(MCD_VAT).Line() + RWA008SGHWOFH1_A_INITIAL_DC);
+
+        /// @notice No dust
+        // VatAbstract(MCD_VAT).file(ilk, "dust", 0)
+
+        /// @notice 3% stability fee
+        JugAbstract(MCD_JUG).file(ilk, "duty", THREE_PCT_RATE);
+
+        /// @notice collateralization ratio 100%
+        SpotAbstract(MCD_SPOT).file(ilk, "mat", RAY); // TODO Should get from RWA team
+
+        /// @notice poke the spotter to pull in a price
+        SpotAbstract(MCD_SPOT).poke(ilk);
+
+        /// @notice give the urn permissions on the join adapter
+        GemJoinAbstract(MCD_JOIN_RWA008SGHWOFH1_A).rely(RWA008SGHWOFH1_A_URN);
+
+        /// @notice set up the urn
+        RwaUrnLike(RWA008SGHWOFH1_A_URN).hope(RWA008SGHWOFH1_OPERATOR);
+
+        /// @notice set up output conduit
+        RwaOutputConduitLike(RWA008SGHWOFH1_A_OUTPUT_CONDUIT).hope(RWA008SGHWOFH1_OPERATOR);
+        /// @notice could potentially kiss some BD addresses if they are available
```

Comment:
We need to whitelist DIIS Group in the conduits as well.
We need to call "mate" on the OutputConduit and InputConduit

Reply:
Done